packages/lit-html/src/lit-html.ts

```diff
@@ -720,7 +716,8 @@ function resolveDirective(
  * An updateable instance of a Template. Holds references to the Parts used to
  * update the template instance.
  */
-class TemplateInstance {
+export type TemplateInstance = Interface<TemplateInstanceImpl>;
```

Comment:
Don't we not need to do this? Is `export type {TemplateInstance}` ok?

Reply:
Reverted